bokeh/models/math_text.py

```diff
@@ -0,0 +1,69 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2021, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+''' Display a mathematics notation from a string value.
+'''
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations
+
+import logging # isort:skip
+log = logging.getLogger(__name__)
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Bokeh imports
+from ..core.properties import Nullable, String
+from ..model import Model
+
+#-----------------------------------------------------------------------------
+# Globals and constants
+#-----------------------------------------------------------------------------
+
+__all__ = (
+    'MathText',
+)
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+class MathText(Model):
+    """
+    Class for indicating that a string value should be interpreted as
+    mathematics notation.
```

Comment:
```suggestion
    Render mathematical content using using LaTeX notation.
```
Suggest making LaTeX a link to https://en.wikipedia.org/wiki/LaTeX